### PR TITLE
Feat/data-label

### PR DESCRIPTION
**Pull request by snenenenenenene**
- Added logic behind the data quality

### CODE REVIEW

File: app/controllers/agenda-items/agenda-item/index.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default class AgendaItemController extends Controller {
 
   showModal = () => {
     this.modalOpen = true;
+    console.log(this.model.agendaItem.agendaItemQuality.qualityMetrics);
```
Comment (aliokan) [inline app/controllers/agenda-items/agenda-item/index.ts:32]: @snenenenenenene Is it needed?

File: app/models/agenda-item.ts [hunk 2]
```diff
@@ -49,6 +49,61 @@ export default class AgendaItemModel extends Model {
   get dateFormatted() {
     return this.session?.dateFormatted;
   }
+
+  get agendaItemQuality(): {
+    qualityMetrics: { label: string; value: boolean }[];
+    qualityValue: number;
```
Comment (aliokan) [inline app/models/agenda-item.ts:55]: @snenenenenenene Is it use somewhere?

**Review by aliokan — COMMENTED** [review]
@snenenenenenene Except the 2 questions it's look good

File: app/controllers/agenda-items/agenda-item/index.ts (hunk 1 above)
Comment (snenenenenenene) [reply app/controllers/agenda-items/agenda-item/index.ts]: WOops, removed it!

File: app/models/agenda-item.ts (hunk 2 above)
Comment (snenenenenenene) [reply app/models/agenda-item.ts]: I've removed the quality value, it was a remnant of what was based on the mock ups. 
But it does seem we won't need it in the near future so I've deleted it!

**Pull request merged**